pkg/common/moerr/error.go

```diff
@@ -473,6 +473,10 @@ func GetOkExpectedEOF() *Error {
 	return &errOkExptededEOF
 }
 
+func New(code uint16, args ...any) *Error {
```

Comment:
Why do you need this one back?

Reply:
You should use Marshal/Unmarshal binary, if you need to transfer errors over wire.    Even though I don't think your code here need to transfer errors over wire -- maybe I missed something.

Reply:
done